src/Entity/Committee.php

```diff
@@ -205,6 +205,13 @@ public function getDescription(): string
         return $this->description;
     }
 
+    public function setDescription(string $description): self
+    {
+        $this->description = $description;
+
+        return $this;
```

Comment:
We don't use the fluent interface on this project.
Please return `void` on setters.